manager.py

```diff
@@ -14,23 +14,29 @@
 from auxiliaries import datetime_from_epoch, set_verbosity
 from sensor import Sensor
 from sender import ServerSender
+from data_handler import Data_Handler
 
 from globalvalues import SIGNAL_PIN, NOISE_PIN
 from globalvalues import POWER_LED_PIN, NETWORK_LED_PIN, COUNTS_LED_PIN
 from globalvalues import DEFAULT_CONFIG, DEFAULT_PUBLICKEY, DEFAULT_LOGFILE
 from globalvalues import DEFAULT_HOSTNAME, DEFAULT_UDP_PORT, DEFAULT_TCP_PORT
 from globalvalues import DEFAULT_SENDER_MODE
 from globalvalues import DEFAULT_INTERVAL_NORMAL, DEFAULT_INTERVAL_TEST
-from globalvalues import DEFAULT_DATALOG
+from globalvalues import DEFAULT_DATALOG, DEFAULT_SEND_TO_SERVER
 from globalvalues import ANSI_RESET, ANSI_YEL, ANSI_GR, ANSI_RED
 
 import signal
 import sys
 
 import csv
+import ast
+from collections import deque
+import os
+
+import socket
 
 def signal_term_handler(signal, frame):
-    print('got SIGTERM')
+    # print('got SIGTERM')
     # If SIGTERM signal is intercepted, the SystemExit exception routines are ran
     sys.exit(0)
 
```

Comment:
You can delete the `CPM_DISPLAY_TEXT` stuff below this, since it is only used in DataHandler now.